infrastructure/metadata/infrastructure/server/context.any.js.ini

```diff
@@ -1,8 +1,22 @@
+[context.any.worker-module.html]
+  expected:
+    if product == "firefox": TIMEOUT # https://bugzilla.mozilla.org/show_bug.cgi?id=1247687
+
 [context.any.sharedworker.html]
   expected:
     if product == "safari" or product == "epiphany" or product == "webkit": ERROR # https://bugs.webkit.org/show_bug.cgi?id=149850
 
+[context.any.sharedworker-module.html]
+  expected:
+    if product == "safari" or product == "epiphany" or product == "webkit": ERROR # https://bugs.webkit.org/show_bug.cgi?id=149850
+    if product == "firefox": TIMEOUT # https://bugzilla.mozilla.org/show_bug.cgi?id=1247687
+
 [context.any.serviceworker.html]
   [context]
     expected:
       if product == "epiphany" or product == "webkit": FAIL # https://bugs.webkit.org/show_bug.cgi?id=200815
+
+[context.any.serviceworker-module.html]
+  expected:
+    if product == "firefox": ERROR
+    if product == "chrome": ERROR # https://crbug.com/1136775
```

Comment:
Hmm, looks like this results in an UNEXPECTED-OK on macOS, where the Chrome version isn't pinned like it is in Taskcluster due to https://github.com/web-platform-tests/wpt/issues/28209.

That'll get increasingly frustrating and I should really fix it. But if you want to fix this now, you could try adding `os == "linux"` as a condition here.

Reply:
Thanks! This is quite mysterious because I expected the test would fail all versions (even on ToT) because https://crbug.com/1136775 isn't fixed yet. Anyway I'll try `os == "linux"` and land this PR, and then later see whether the test actually fails when imported to Chromium repository.

Reply:
Er, I found https://crbug.com/1136775 was partially fixed so now I think it's reasonable that the test passes on Chrome Dev.